### DISCUSSION TITLE
resetting  meshcentral to use default  ports

### DISCUSSION THREAD

**Discussion by harmelgill** [Ylianst/MeshCentral#6109] [Q&A]
Hi  i would like  to access  meshcentral  from internet using ports 80 /443

currently i can only access using port 444 
this is  my current json file 
 
 {
  "$schema": "http://info.meshcentral.com/downloads/meshcentral-config-schema.json",
  "__comment1__": "This is a simple configuration file, all values and sections that start with underscore (_) are ignored. Edit a section and remove the _ in front of the name. Refer to the user's guide for details.",
  "__comment2__": "See node_modules/meshcentral/sample-config-advanced.json for a more advanced example.",
  "settings": {
    "cert": "xxxxxxxxx.com",
    "_WANonly": true,
    "_LANonly": true,
    "_sessionKey": "MyReallySecretPassword1",
    "CookieIpCheck": false,
    "port": 80,
    "aliasPort": 443,
    "port": 443,
    "redirPort": 80,
    "redirPort": 80,
    "redirAliasPort": 80
  },
  "domains": {
    "": {
      "title": "test",
      "_title2": "Servername",
      "_minify": true,
      "_newAccounts": true,
      "_userNameIsEmail": true
    }
  },
  "letsencrypt": {
    "__comment__": "Requires NodeJS 8.x or better, Go to https://letsdebug.net/ first before trying Let's Encrypt.",
    "email": "xxxxxxxx@gmail.com",
    "names": "xxxxxxxxx.com",
    "_production": false
  }
}

**Comment by si458** [comment]
If ur meshcentral is starting up and saying its using port 444
Then it will be something on ur machine/server is running on port 443
So u need to identify what it is then stop that service, then restart meshcentral
